jwst/outlier_detection/outlier_detection.py

```diff
@@ -392,39 +375,7 @@ def flag_cr(sci_image, blot_image, gain_image, readnoise_image, **pars):
     blot_data = blot_image.data * exptime
     blot_deriv = abs_deriv(blot_data)
 
-    # This mask can take into account any crbits values
-    # specified by the user to be ignored.
-    # dq_mask = build_mask(sci_image.dq, CRBIT)
```

Comment:
I see that the build_mask function is still included in the code, but it isn't used anywhere. Will it be in the future?

Reply:
That function has now been removed.  We have another copy of it in the resample package if needed by any JWST code.